children/README.md

```diff
@@ -0,0 +1,214 @@
+The `child` link relations in a Catalog already allow for describing these
+relationships, but require a client to retrieve each resource URL to find any information about the children
+(e.g., title, description), which can cause significant performance issues in user-facing applications.
+
+It is recommended that the Catalog and Collection objects returned by the `/children` endpoint are the same
+as those referenced by `child` link relations in the STAC API Landing Page. Following these semantics, it should
+return the immediate children of the root Catalog, rather than all descendant catalogs or collections.
+
+## Link Relations
+
+This conformance class also requires implementation of the link relations in the [STAC API - Core](../core) conformance class.
+
+The following Link relations shall exist in the Landing Page (root).
+
+| **rel**    | **href**    | **From**        | **Description**                  |
+| ---------- | ----------- | --------------- | -------------------------------- |
+| `children` | `/children` | STAC API - Children | List of children of this catalog |
+
+The following Link relations shall exist in the `/children` endpoint response.
+
+| **rel** | **href**    | **From**            | **Description** |
+| ------- | ----------- | ------------------- | --------------- |
+| `root`  | `/`         | STAC Core           | The root URI    |
+| `self`  | `/children` | STAC API - Children | Self reference  |
+
+## Endpoints
+
+This conformance class also requires for the endpoints in the [STAC API - Core](../core) conformance class to be implemented.
+
+| Endpoint    | Returns        | Description                                                                                                            |
+| ----------- | -------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `/children` | JSON           | Object with a list of child Catalogs and Collections                                                                   |
+
+STAC APIs implementing the `STAC API - Children` conformance class must support HTTP GET operation at
+`/children`, with the return JSON document consisting of an array of all child Catalogs and Collections in a field `children` and an 
+array of Links in a field `links`.
+
+## Pagination
+
+The `/children` endpoint supports a pagination mechanism that aligns with pagination as described in the 
+OGC API - Common - Part 2: Geospatial Data specification. This is described in detail in
+the [STAC - Features Collection Pagination section](../ogcapi-features/README.md#collection-pagination).
+To the greatest extent possible, the catalog should be structured such that all children can be
+retrieved from the endpoint in a single call.
+
+## Example
+
+Below is a minimal example, but captures the essence. Each object in the `children` array 
```

Comment:
nice example!

Reply:
So if an API contains the 'children' conformance class, does this mean that any sub-catalog within the tree have a 'children' link? I think I'm hung up on "shall contain" verbiage....I looked it up and in terms of a legal definition it's stated to be ambiguous as it can mean "may, will or must."

Reply:
I can't find the exact link right now, but OGC follows another standards body's definitions (I think ISO) where shall and must mean the same thing (you have to do this) and "may" means it's not prohibited but you don't have to. 

sub-catalogs within the tree that are also STAC APIs don't have to implement children -- each sub-catalog/sub-API is independent of the parent's conformance.